dom-events.tar/dom-events/buttons.js

```diff
@@ -13,7 +13,7 @@ function handleMouseOver(event){
   console.log(event);
   console.log(event.target);
 }
- const element =document.querySelector('.hover-button');
+ const element2 =document.querySelector('.hover-button');
   if (!$hoverButton) throw new Error('The $hoverButton query failed');
   element.addEventListener('mouseover',event)
```

Comment:
make sure you're adding the event listener to the right variable now that you've changed the name of the `element` on line 16